setup.py

```diff
@@ -51,7 +51,7 @@
         "autobuild": ["chromadb", "sentence-transformers", "huggingface-hub"],
         "teachable": ["chromadb"],
         "lmm": ["replicate", "pillow"],
-        "graphs": ["networkx~=3.2.1", "matplotlib~=3.8.1"],
+        "graphs": ["networkx", "matplotlib"],
```

Comment:
Consider documenting this optional dependency in https://microsoft.github.io/autogen/docs/installation/Optional-Dependencies

Reply:
Done

Reply:
I didn't see the change. Is it in a different PR?

Reply:
https://github.com/microsoft/autogen/blob/GraphGroupChat/setup.py#L54

Excuse me, is it as expected?

Reply:
I meant the documentation in docs/installation/Optional-Dependencies